### PR TITLE
LPS-105883 Implement list view for account roles

### PR DESCRIPTION
**Pull request by pei-jung**
/cc @drewbrokke @alee8888

### CODE REVIEW

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/AccountRoleDisplay.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		return _role.getTitle(locale);
+	}
+
+	public long getRoleId() {
+		return _role.getRoleId();
+	}
+
+	private AccountRoleDisplay(AccountRole accountRole) throws Exception {
+		_accountRole = accountRole;
+		_role = RoleLocalServiceUtil.getRole(accountRole.getRoleId());
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/AccountRoleDisplay.java]: @pei-jung could we use `accountRole.getRole()` here instead?

Comment (pei-jung) [reply modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/AccountRoleDisplay.java]: Ok, will update and resend

**Comment by pei-jung** [comment]
Closing to make the changes per Drew's comment. Will resend.

**Comment by brianchandotcom** [comment]
@pei-jung see a6f8286a8573b6a234e089b5c01abda5bc0d30b0

Can you also send a follow up pull and move this out of EditAccountRoleMVCActionCommand.java:

```
		String[] invalidCharacters = StringUtil.split(
			RoleConstants.NAME_INVALID_CHARACTERS, StringPool.SPACE);

		if ((!PropsValues.ROLES_NAME_ALLOW_NUMERIC &&
			 Validator.isNumber(name)) ||
			RoleConstants.NAME_RESERVED_WORDS.equals(name) ||
			(StringUtil.indexOfAny(name, invalidCharacters) > -1)) {

			throw new RoleNameException();
		}
```

I generally like to move validation logic into a reusable class, and not in a *MVC*Command.java

@drewbrokke I think I had you do a similar extraction not too long ago. Can you discuss with @pei-jung thx

**Comment by pei-jung** [comment]
Follow up pull sent here https://github.com/brianchandotcom/liferay-portal/pull/84430

**Pull request closed without merging**